spec/requests/authentications_spec.rb

```diff
@@ -258,13 +258,12 @@
             'task_id' => a_kind_of(String)
           ),
           a_hash_including(
-            'success' => true,
-            'message' => 'Creating Authentication',
-            'task_id' => a_kind_of(String)
+            'success' => false,
+            'message' => 'must supply a manager resource'
           )
         ]
       }
-      post(api_authentications_url, :params => { :resources => [create_params, create_params] })
+      post(api_authentications_url, :params => {:resources => [create_params, create_params.except(:manager_resource)]})
```

Comment:
why is this needed in this PR ?

Reply:
deleted it... remembered why it is there.

I wanted to ensure that if we have a successful and a failed creation of an authentication (it requires a manager_resource to create the record) that it would still return a 200.